_dev/coding-style.md

```diff
@@ -6,8 +6,65 @@ description: The preferred style for code committed to NeoMutt
 
 # Coding Style
 
-Every project has its own coding style, this one is no exception.
-However, I haven't written down the rules.  For now, ...
+## at the moment (pre-fork)
 
-- Trailing whitespace is **evil**
+Generally speaking, we don't want to diverge much from upstream mutt. We're
+still merging patches in - and if we didn't follow upstream's coding style,
+merging would be much harder.
```

Comment:
well, that's the current state of mutt's codebase, so we'd be only improving by having a style guide and then at a later point reformat the whole source and have it all homogeneous.